model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/nh_solve/solve_nonhydro.py

```diff
@@ -1487,48 +1448,33 @@ def run_corrector_step(
             offset_provider={},
         )
 
-        start_cell_lb_plus2 = self.grid.get_start_index(
-            dims.CellDim, h_grid.HorizontalMarkerIndex.lateral_boundary(dims.CellDim) + 2
+        cell_domain = h_grid.domain(dims.CellDim)
+        edge_domain = h_grid.domain(dims.EdgeDim)
+        start_cell_lb_level_3 = self.grid.start_index(
```

Comment:
More consistency renaming
```suggestion
        start_cell_lateral_boundary_level_3 = self.grid.start_index(
```